python/lsst/ts/xml/data/sal_interfaces/ESS/ESS_Events.xml

```diff
@@ -1,61 +1,69 @@
 <SALEventSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
+  <Enumeration>AlarmLevel_Red, AlarmLevel_Orange, AlarmLevel_Yellow, AlarmLevel_Green</Enumeration>
   <SALEvent>
     <Subsystem>ESS</Subsystem>
     <EFDB_Topic>ESS_logevent_lightningStrike</EFDB_Topic>
-    <Description>Lightning strikes and high electrical field warnings.</Description>
+    <Description>Lightning strike, if detected.</Description>
     <item>
       <EFDB_Name>sensorName</EFDB_Name>
-      <Description>The name of the sensor.</Description>
+      <Description>The user-defined name of the sensor (e.g. AuxTel LD-250).</Description>
       <IDL_Type>string</IDL_Type>
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>strike</EFDB_Name>
-      <Description>Lighting strike alarms, as a mask with numStrikeBits bits. 1 means an alarm is active. Higher-order bits are for strikes that are closer, and thus more worrisome. Each alarm will eventually reset itself if no further activity in that alarm range is detected.</Description>
-      <IDL_Type>unsigned short</IDL_Type>
-      <IDL_Size>1</IDL_Size>
-      <Units>unitless</Units>
+      <EFDB_Name>correctedDistance</EFDB_Name>
+      <Description>Distance of the strike, corrected by the hardware.</Description>
+      <IDL_Type>short</IDL_Type>
+      <Units>km</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>highField</EFDB_Name>
-      <Description>High electrical field alarms, as a mask with numHighFieldBits bits. 1 means an alarm is active. Higher-order bits are for larger electrical fields, and thus are more worrisome. Each alarm will eventually reset itself if no further activity in that alarm range is detected</Description>
-      <IDL_Type>unsigned short</IDL_Type>
-      <IDL_Size>1</IDL_Size>
-      <Units>unitless</Units>
+      <EFDB_Name>uncorrectedDistance</EFDB_Name>
+      <Description>Distance of the strike, not corrected by the hardware.</Description>
+      <IDL_Type>short</IDL_Type>
+      <Units>km</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>numStrikeBits</EFDB_Name>
-      <Description>The maximum number of strike bits.</Description>
-      <IDL_Type>unsigned short</IDL_Type>
-      <IDL_Size>1</IDL_Size>
-      <Units>unitless</Units>
+      <EFDB_Name>bearing</EFDB_Name>
+      <Description>Bearing of the strike.</Description>
+      <IDL_Type>short</IDL_Type>
+      <Units>degree</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>numHighFieldBits</EFDB_Name>
-      <Description>The maximum number of high field bits.</Description>
-      <IDL_Type>unsigned short</IDL_Type>
-      <IDL_Size>1</IDL_Size>
+      <EFDB_Name>level</EFDB_Name>
+      <Description>Alarm level, an AlarmLevel enum limited to Red, Orange and Yellow.</Description>
```

Comment:
I hope you plan to indicate "no strike" in this event when the ESS starts up and some "reasonable" time after the most recent strike is detected. (The lightning system consolidating box does this and it's generally a good idea).

If you can have more than one strike event active at the same time then you probably need some kind of ID field so we can tell the difference between each strike event.

Reply:
I wonder if you should bother with a level? It seems like we could determine our own severity directly from the data -- the Watcher will probably have a rule to handle this. Or does this level come from the sensor?

Reply:
Good points all of them. I have removed the enum plus the items referring to them and adjusted the descriptions.